classes/SnippetLoader.php

```diff
@@ -0,0 +1,181 @@
+<?php namespace StudioAzura\W2C\Classes;
```

Comment:
Incorrect namespace here :)
```suggestion
<?php namespace Winter\Pages\Classes;
```